src/core/geometry/mesh/Unstructured.cpp

```diff
@@ -7,10 +7,14 @@ namespace mesh {
 
 Unstructured::Unstructured(const CoordR3Group& cG,
                            const ElemRGroup& elem,
-                           const LayerGroup& layers) :   
+                           const LayerGroup& layers,
+                           const std::vector<junction::Junction>& junctions,
```

Comment:
This is a very big change... Is not any other way of doing this?. For instance, maybe we could think on bundles and junctions as elements which would be added to the element group? If we incorporate this here we are opening a whole new world of possibilities.

Reply:
Removed Bundles from Unstructured in commit e0876b681447e7be6b8220111352cf1fac6c7c78. Junctions would need more talk to see how we proceed.